donation/serializers.py

```diff
@@ -61,3 +33,49 @@ def create(self, validated_data):
             search_slug=self.switch(validated_data['blood_group'])
         )
         return donation_request
+
+    def update(self, instance, validated_data):
+        instance.blood_group = validated_data.get(
+            'blood_group', instance.blood_group)
+        instance.quantity = validated_data.get('quantity', instance.quantity)
+        instance.location = validated_data.get('location', instance.location)
+        instance.priority = validated_data.get('priority', instance.priority)
+        instance.description = validated_data.get(
+            'description', instance.description)
+        instance.document = validated_data.get('document', instance.document)
+        if(validated_data.get('is_complete', False) is True):
+            instance.in_progress = False
+        instance.is_complete = validated_data.get(
+            'is_complete', instance.is_complete)
+        instance.save()
+        return instance
+
+
+class DonationGetSerializer(serializers.ModelSerializer):
```

Comment:
why this name? this doesn't give me an idea, name should tell exactly why we need this